nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-framework-components/src/main/java/org/apache/nifi/controller/StandardProcessorNode.java

```diff
@@ -1969,17 +2033,12 @@ public void setMaxBackoffPeriod(String maxBackoffPeriod) {
     }
 
     @Override
-    public String evaluateSchedulingPeriodFromParameter(final String schedulingPeriod) {
+    public String evaluateParameter(final String parameter) {
```

Comment:
I don't think the argument here is really a parameter is it? It's some value that may or may not contain parameters. Perhaps should call it `evaluateParameters` and call the argument `value`?